en/docs/chapter_tree/binary_tree.md

```diff
@@ -221,13 +221,13 @@ The commonly used terminology of binary trees is shown in the following figure.
 
 !!! tip
 
-    Please note that we usually define "height" and "depth" as "the number of edges passed," but some problems or textbooks may define them as "the number of nodes passed." In this case, both height and depth need to be incremented by 1.
+    Please note that we typically define "height" and "depth" as "the number of edges traversed," but some problems or textbooks may define them as "the number of nodes traversed." In such cases, both height and depth need to be incremented by 1.
 
 ## Basic operations of binary trees
 
 ### Initializing a binary tree
 
-Similar to a linked list, initialize nodes first, then construct references (pointers).
+Similar to a linked list, begin by initialize nodes first, then construct references (pointers).
```

Comment:
"begin by initialize nodes first" can be "begin by initializing nodes", since the combination of "begin" and "first" is kind of redundant.

Reply:
er... my bad. ill change it now.